Source/WebKit/NetworkProcess/NetworkSession.cpp

```diff
@@ -89,9 +85,6 @@ std::unique_ptr<NetworkSession> NetworkSession::create(NetworkProcess& networkPr
 #if USE(CURL)
     return NetworkSessionCurl::create(networkProcess, parameters);
 #endif
-#if PLATFORM(HAIKU)
-    return NetworkSessionHaiku::create(networkProcess, parameters);
```

Comment:
In theory it's possible to switch between curl and libnetapi for the network support. We may switch back to libnetapi someday (it seems pretty unlikely, but you never know).

So, even if this ends up being mostly unused, it would be nice to keep the files for libnetapi around in some way (even if not included in the build).

Reply:
Ok, I was wondering about that. In that case, should we also keep the Haiku-specific implementation of shared memory around too?